tmt/steps/prepare/ansible.py

```diff
@@ -54,20 +71,21 @@ def options(cls, how=None):
                 help='Optional arguments for ansible-playbook.')
             ] + super().options(how)
 
-    def default(self, option, default=None):
+    def default(self, option: str, default: Optional[Any] = None) -> Any:
         """ Return default data for given option """
         if option == 'playbook':
             return []
         return default
 
-    def wake(self, keys=None):
+    # TODO: use better types once superclass gains its annotations
+    def wake(self, keys: Optional[List[str]] = None):
```

Comment:
Why are tests not complaining about this method lacking a return type annotation??

Reply:
Aha, got it.

@guy9050 please, update `mypy.ini`, add `ansible.py` to the list of files mypy's allowed to check (https://github.com/teemtee/tmt/blob/main/mypy.ini#L10)